packages/server/src/user/user.service.ts

```diff
@@ -191,5 +216,29 @@ export class UserService {
     }
   }
 
+  /**
+   * Update user's role
+   *
+   * @param id uuid of the user as string
+   * @param roleToEdit role needs to edit in number representation, refer to `role.enum.ts`
+   * @param addRole `true` for add new role to user, `false` for remove role from user
+   */
+  async updateUserRole(id: string, roleToEdit: number, addRole: boolean): Promise<void> {
```

Comment:
Create a default to add a Role to be true. Then in the controller you just need to set it to false to remove the role